.pre-commit-config.yaml

```diff
@@ -96,5 +96,5 @@ repos:
         name: "Bandit security check"
         language: system
         types: [file, python]
-        exclude: ^kedro-(telemetry|docker|datasets|airflow)/tests/|^kedro-(telemetry|docker|datasets|airflow)/features/
+        exclude: ^kedro-(telemetry|docker|datasets|airflow)/tests/|^kedro-(telemetry|docker|datasets|airflow)/features/|kedro-datasets/kedro_datasets_experimental/tests/
```

Comment:
Maybe we use `# nosec B108 ` for the target line instead?

Reply:
let me try to fix this